utils/build-mlir-air.sh

```diff
@@ -54,7 +53,7 @@ cmake .. \
     -DCMAKE_C_COMPILER=clang \
     -DCMAKE_CXX_COMPILER=clang++ \
     -DCMAKE_INSTALL_PREFIX="../${INSTALL_DIR}" \
-    -DCMAKE_TOOLCHAIN_FILE_OPT=${CMAKEMODULES_DIR}/toolchain_clang_crosscomp_arm_petalinux.cmake \
+    -DCMAKE_TOOLCHAIN_FILE_OPT=${CMAKEMODULES_DIR}/cmakeModulesXilinx/toolchain_clang_crosscomp_arm_petalinux.cmake \
```

Comment:
I think we can remove this line. We don't have the `CMAKE_TOOLCHAIN_FILE_OPT` option anymore.

Reply:
Agreed. This line is now removed. Thanks.